tests/py-flux-test.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+from flux.thermal import PccThermalModel1D
+from flux.grids import setgrid
+
+class PyFluxTest(unittest.TestCase):
+    def test_conductionQ(self):
+
+        # import template output
+        df_template = pd.read_csv("/home/sberton2/Works/NASA/radiosity/python-flux/tests/template.out",sep='\s+').iloc[:,:-1]
```

Comment:
fix hardcoded path to template.out

Reply:
ops, yes, I was a bit brutal there, sorry ^^